contracts/Deployer.sol

```diff
@@ -58,7 +71,8 @@ contract Deployer is Destructible, GroupAdmin {
                 _coolingPeriod,
                 msg.sender,
                 _tokenAddress,
-                clearFee
+                clearFee,
+                address(ct)
```

Comment:
Instead of passing the conference ticket address, we could just pass baseTokenUri

Reply:
With the described limitation the conference ticket address is mandatory.